agent/src/beerocks/slave/platform_manager/platform_manager_thread.cpp

```diff
@@ -208,7 +208,7 @@ static bool fill_platform_settings(
     msg->platform_settings().certification_mode = uint8_t(platform_common_conf.certification_mode);
     msg->platform_settings().stop_on_failure_attempts =
         uint8_t(platform_common_conf.stop_on_failure_attempts);
-    msg->platform_settings().backhaul_max_vaps = uint8_t(platform_common_conf.backhaul_max_vaps);
+    db->device_conf.back_radio.backhaul_max_vaps = uint8_t(platform_common_conf.backhaul_max_vaps);
```

Comment:
```suggestion
    db->device_conf.back_radio.backhaul_max_vaps = static_cast<uint8_t>(platform_common_conf.backhaul_max_vaps);
```

This PR seems like a good opportunity to replace all these C style casts with the preferred C++ style casts